front/package.json

```diff
@@ -183,7 +183,7 @@
     "@types/react-transition-group": "4.4.1",
     "@types/segment-analytics": "0.0.33",
     "@types/shelljs": "0.8.8",
-    "@types/styled-components": "5.1.7",
+    "@types/styled-components": "^5.1.7",
```

Comment:
Normally we work with fixed versions. I also see there's been a similar change to react-select-event's version that should be undone. Could you also do that in this PR?

Reply:
Hey, Brent! Don't mind changing this but I have a question - what is the reason for this practice?
The versions are locked in the package-lock file so there shouldn't be any discrepancies.

Reply:
Good question. From what I remember, we are doing it to not introduce breaking changes unexpectedly, but ^ doesn't do that. @DavidCnoops any other reason why we are doing this?

Reply:
@brentguf Thanks for the review! 
As the PR is approved, I am merging as is. Happy to continue this discussion and change the version of this package in a separate pr, if there is a need 🚀